src/app/index.ts

```diff
@@ -276,42 +243,54 @@ module.exports = yo.extend({
   writing: {
     copyFiles: function () {
       try {
-        let language = this.project.ts ? 'ts' : 'js';
+        let language = this.project.scriptType  === typescript ? 'ts' : 'js';
 
         /** Show type of project creating in progress */
-        if (this.project.framework !== 'manifest-only') {
+        if (!this.project.isManifestOnly && !this.project.isExcelFunctionsProject) {

```

Comment:
The logic would be better reversed:

if (isManifestOnly) {
   …
} else if (isExcelCustomFunctionsProject) {
  ….
} else {
  …
}

This is why it is better to make if statements use positive logic, so you avoid double-negatives.  :-)